livox_ros_driver/livox_ros_driver/lddc.cpp

```diff
@@ -683,9 +683,6 @@ ros::Publisher *Lddc::GetCurrentPublisher(uint8_t handle) {
           name_str, queue_size);
     }

```

Comment:
Add handling for an invalid `transfer_format_`.

Reply:
Done.